docs/_includes/Running-Flink-on-Alluxio/flink-profile-build.md

```diff
@@ -1,3 +1,3 @@
 ```bash
-$ mvn clean package -Pflink -DskipTests
+$ mvn clean package -DskipTests
```

Comment:
You also need to update the description in the docs, where mentions flink profile.